defaults.reek

```diff
@@ -44,6 +44,9 @@ LongYieldList:
 ManualDispatch:
   enabled: true
   exclude: []
+MissingSafeMethod:
```

Comment:
This trips up codeclimate at the moment. I'm not sure what we should do. This would probably be an issue whenever we add a new smell anyway.